_universum/modules/vcs/github_vcs.py

```diff
@@ -44,6 +44,18 @@ def __init__(self, *args, **kwargs):
 
                     In CI builds commit ID is usually extracted from webhook and handled automatically.  
                 """)
+        utils.check_required_option(self.settings, "token", """
+                    github api token is not specified.
+
+                    GitHub API token is used for application authorization. Expected format is
+                    'v1.4e96f8c2d1922c3b154a65ca7ecb91f6994fb0c5'.
+                    Please specify the token by using '--github-token' ('-ght')
+                    command line parameter or by setting GITHUB_TOKEN environment variable.
+
+                    In CI builds github api token is expected to be acquired automatically before build.
+                    For information on how to acquire the token please see
```

Comment:
This reference wouldn't help the user.
We have to describe github integration in our docs and provide the link in the error message.